src/app/contexts/Ledger/use-ledger.ts

```diff
@@ -0,0 +1,182 @@
+import { Coins } from "@arkecosystem/platform-sdk";
+import { Profile, WalletFlag } from "@arkecosystem/platform-sdk-profiles";
+import { BigNumber } from "@arkecosystem/platform-sdk-support";
+import Transport from "@ledgerhq/hw-transport";
+import retry from "async-retry";
+import { useCallback, useEffect, useMemo, useReducer, useRef } from "react";
+
+import { useEnvironmentContext } from "../Environment";
+import { defaultLedgerState, ledgerStateReducer } from "./Ledger.state";
+
+const formatDerivationPath = (coinType: number, index: number) => `44'/${coinType}'/${index}'/0/0`;
```

Comment:
I don't know the requirements or anything regarding this PR (I just glanced through it), but shouldn't the index be at the last part as specified in the bip44 standard? The "index" currently changes the account, not the address_index, or was this the intention? cc @faustbrian

Reply:
Hey @DalderupMaurice thanks for letting me know, @sleepdefic1t had already contacted me about this and we will fix it.